integration/default_test.go

```diff
@@ -163,5 +163,46 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 				),
 			)
 		})
+
+		it("builds an oci image with poetry (dependency management only, not runnable)", func() {
+			var err error
+			source, err = occam.Source(filepath.Join("testdata", "poetry-dep-only"))
```

Comment:
I think that `poetry-dep-only` should be renamed to `poetry` because the dep-only is implied. More specifically - this buildpack will not be expanded to support `poetry-run` (as this buildpack is mutually-exclusive with `poetry-run`) so there will never be the "non-dep-only" use-case.

Reply:
done